packages/rest/src/rest.server.ts

```diff
@@ -234,6 +234,20 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
+  /**
+   * Apply express settings. Bind functions to the server context (not express).
+   */
+  protected _applyExpressSettings() {
+    Object.entries(this.config.expressSettings || {}).forEach(
+      ([key, value]) => {
+        if (typeof value === 'function') {
+          value = value.bind(this);
```

Comment:
Why `bind` here? I think it's the responsibility of the code that provides the configuration to supply the correct function.

Reply:
not sure, I was thinking it's better than having express bound to this. Removed it.

Reply:
BTW, we need to keep the default `'query parser': 'extended'` if not present. I fixed it in my clone (to be pushed back to the PR)